src/Makefile.am

```diff
@@ -410,10 +410,10 @@ unicode_parser_SOURCES =        \
     unicode-parser.c            \
     $(NULL)
 unicode_parser_CFLAGS =         \
-    $(GLIB2_CFLAGS)             \
+    @GLIB2_CFLAGS@             \
     $(NULL)
 unicode_parser_LDADD =          \
-    $(GLIB2_LIBS)               \
+    @GLIB2_LIBS@               \
```

Comment:
Since I cannot comment on the git commit, I moved my comment into the patch:

Should we have those changes?
Sorry, I don't understand the demerit not to integrate this change. I think users can modify *_CFLAGS and *_LIBS in each Makefile currently without running automake & config.status again.

Reply:
Surely no one has wanted to override `GLIB2_CFLAGS`, `GOBJECT2_CFLAGS`, `GLIB2_LIBS`, `GOBJECT2_LIBS` *only* for `emoji_parser` and `unicode_parser` without rerunning `./configure`. That makes no sense.

I believe the usage of `$(...)` here was simply a mistake in

````
commit 160d3c975af91eea6b8271b757be769b8ceef98d
Author: fujiwarat <takao.fujiwara1@gmail.com>
Date:   Tue Jun 21 18:10:21 2016 +0900

    engine: Implement Emoji typing with XKB engines
    
    Now Ctrl+Shift+e can convert an Emoji annotation to the Emoji characters
    likes Ctrl+Shift+u.
    The annotations are described as "Keywords" in the Unicode Emoji Data:
    http://www.unicode.org/emoji/charts/emoji-list.html
    'emoji-parser' compiles 'emoji-list.html' and generates 'emoji.dict'
    
    Review URL: https://codereview.appspot.com/295610043
````

This is simply not how you specify these variables. They are provided by autoconf substitution, which is done with `@VARNAME@`, as is done throughout the rest of the project's `Makefile.am`s, including elsewhere in this `src/Makefile.am`.

Reply:
It was not my mistake but the expected change. I had the plan to change all @foo@ to $(foo) by commit.
E.g. modifying GLIb2_LIBS to indicate BUILD/glib/glib/.libs in Makefile only is useful for me the debugging.

I don't find the explicit demerit but I can follow you if you strongly recommend your patch.